config_lib/parsers/parser_yaml.py

```diff
@@ -1,11 +1,207 @@
+import re
+from datetime import datetime
+
+
 class YAMLSyntaxError(Exception):
+
     def __init__(self, message, line_num):
         super().__init__(f"YAML Syntax Error on line {line_num}: {message}")
         self.line_number = line_num
```

Comment:
Remove the blank line inside the class definition. It’s better to do it like this:
```suggestion
class YAMLSyntaxError(Exception):
    def __init__(self, message, line_num):
        super().__init__(f"YAML Syntax Error on line {line_num}: {message}")
        self.line_number = line_num
```